crates/rust-analyzer/src/semantic_tokens.rs

```diff
@@ -62,6 +62,7 @@ pub(crate) const SUPPORTED_MODIFIERS: &[SemanticTokenModifier] = &[
 #[derive(Default)]
 pub(crate) struct ModifierSet(pub(crate) u32);
 
+#[allow(clippy::suspicious_op_assign_impl)]
```

Comment:
We don’t add explicit allows for clippy

Reply:
How should this be handled then? It causes clippy to exit.

Reply:
Report a bug against clippy :) and blacklist the lint in cargo xtask lint

Reply:
Ah I missed xtask lint.